client/src/components/App.jsx

```diff
@@ -117,24 +120,27 @@ class App extends React.Component {
 
     getGalleryData() {
         axios.get('/gallery')
-        .then(({data})=> this.setState({
-            images: data,
-        }))
+        .then(({data})=> this.addIndex(data))
```

Comment:
please leave space before and after arrow
```suggestion
        .then(({data}) => this.addIndex(data))
```